src/Compilers/CSharp/Test/Syntax/Parsing/CollectionExpressionParsingTests.cs

```diff
@@ -6992,28 +6992,17 @@ public void TestError5()
         UsingExpression("[...e]",
             // (1,2): error CS8635: Unexpected character sequence '...'
             // [...e]
-            Diagnostic(ErrorCode.ERR_TripleDotNotAllowed, "").WithLocation(1, 2),
-            // (1,4): error CS1525: Invalid expression term '.'
-            // [...e]
-            Diagnostic(ErrorCode.ERR_InvalidExprTerm, ".").WithArguments(".").WithLocation(1, 4));
+            Diagnostic(ErrorCode.ERR_TripleDotNotAllowed, "").WithLocation(1, 2));
```

Comment:
nicer parsing than before.  we treat this as a slightly miswritten spread inside a collection expr.